event_log.c

```diff
@@ -204,9 +215,12 @@ int log_event_tpm20(u32 pcr, u8 sha1[20], u8 sha256[32], char *event)
 int event_log_init(struct tpm *tpm)
 {
     unsigned int min_size;
-    struct skl_tag_evtlog *t = next_of_type(&bootloader_data, SKL_TAG_EVENT_LOG);
+    struct slr_entry_log_info *info;
+    u8 hash[SHA1_DIGEST_SIZE];
 
-    if ( t == NULL || next_of_type(t, SKL_TAG_EVENT_LOG) != NULL )
+    info = next_entry_with_tag(&bootloader_data.entries, SLR_ENTRY_LOG_INFO);
```

Comment:
For consistency, use `NULL` here as well.
```suggestion
    info = next_entry_with_tag(NULL, SLR_ENTRY_LOG_INFO);
```

Reply:
https://github.com/TrenchBoot/secure-kernel-loader/compare/f2a18cb10d6548d177ca1eff9b8334344b2a306b..b9aaa5c37b91adeed72a067dde349129b9970fde#diff-7aae9b14e1b9cc986e1cf987c539eebb9634a4bb0d1d55d59ec0f65a15fe82ccR222